libraries/abstractions/common_io/test/iot_test_common_io.c

```diff
@@ -57,23 +57,23 @@ TEST_GROUP_RUNNER( Common_IO )
     size_t i = 0;
 
     #if ( IOT_TEST_COMMON_IO_UART_SUPPORTED == 1 )
-        for( i = 0; i < UART_TEST_SET; i++ )
+        for( i = 0; i < IOT_TEST_COMMON_IO_UART_SUPPORTED; i++ )
```

Comment:
Thanks for doing this.

Reply:
With such change: for( i = 0; i < IOT_TEST_COMMON_IO_UART_SUPPORTED; i++ )
the loop will be actually always 1, so do we still need foor loop here.

Reply:
No, this same macro will mean how many tests you want to run, the same meaning of the current `UART_TEST_SET`

Reply:
But we have #if ( IOT_TEST_COMMON_IO_UART_SUPPORTED == 1 ), right?
We only check if equals 1, if not 1, it won't run at all.

Reply:
Good catch, I'll do the needed change.

Reply:
Fixed, please check.